docs/source/tutorial/add_another_pipeline.md

```diff
@@ -520,5 +520,11 @@ kedro run --runner=module.path.to.my.runner
 
 `ParallelRunner` performs task parallelisation via multiprocessing, while `ThreadRunner` is intended for use with remote execution engines such as [Spark](../integrations/pyspark_integration.md) and {class}`Dask<kedro-datasets:kedro_datasets.dask.ParquetDataset>`.
 
+### Watch the video
+
+```{eval-rst}
+..  youtube:: _B6R2uOj3-s
+    :width: 100%
+```
```

Comment:
There's a number of videos in the course corresponding to this page. This is the one corresponding to runners (good), there's another one corresponding to namespaces (titled in this page "Modular pipelines"[sic]) https://www.youtube.com/watch?v=cYHHVAoWZ2E, and a couple more. But I don't think we want to litter the page with videos. What about including the namespaces one? Runners aren't really that complicated

Reply:
Let's revisit that when we revise the modular pipelines docs as I suspect it may be better placed there (and likewise the tutorial guidance on namespaces/modular pipelines will need update). Agree runners aren't that complicated so I'll remove that video on the basis that people can find it anyway if they need it.